language-bindings/rust/src/instance.rs

```diff
@@ -0,0 +1,94 @@
+//! an instantiated module. The module is instantiated with the given imports.
+//!
+
+use ::core::ffi::c_char;
+
+use wamr_sys::{wasm_module_inst_t, wasm_runtime_deinstantiate, wasm_runtime_instantiate};
+
+use crate::{
+    helper::error_buf_to_string, helper::DEFAULT_ERROR_BUF_SIZE, module::Module, RuntimeError,
+};
+
+#[derive(Debug)]
+pub struct Instance {
+    instance: wasm_module_inst_t,
+}
+
+impl Instance {
+    pub fn new(module: &Module, stack_size: u32) -> Result<Self, RuntimeError> {
```

Comment:
Had better add `heap_size: u32` or `host_managed_heap_size: u32` also, like wasm_runtime_instantiate, to allow developer to run the `nostdlib` wasm module. We may add some comments to help developer to pass the correct values, but had better not ignore it.